dump25/sitelinks/text.py

```
@@ -0,0 +1,335 @@
+        # ---
+        family_name = family
+        # ---
+        if families_names.get(family, family) != family:
+            family_name = family + " / " + families_names.get(family, family)
+        # ---
+        if family == "others":
+            family_section = f"== {family_name} ==\n"
+        else:
+            family_section = f"=== {family_name} ===\n"
+        # ---
+        family
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)